sunpycube/cube/cube_utils.py

```diff
@@ -425,19 +427,17 @@ def convert_point(value, unit, wcs, axis, _source='cube'):
         value = value.value
         unit = value.unit
     if _source == 'cube':
-        wcsaxis = -1 - axis if wcs.oriented or not wcs.was_augmented \
+        wcsaxis = -1 - axis if wcs.oriented and not wcs.was_augmented \
```

Comment:
it should be and as wcs.was_augmented = false means, wcs object has dimension 3 so wcsaxis = -1-axis not -2-axis and similary for all cases.